Sources/Hyperspace/HTTP/HTTP.swift

```diff
@@ -162,6 +162,9 @@ public struct HTTP {
 
         /// The `URL` to which the `Request` originated.
         public let url: URL?
+        
+        /// The HTTP method for a request
```

Comment:
Please add a period at the end for consistency with other comments.
```suggestion
        /// The HTTP method for a request.
```